src/data/data-format.coffee

```diff
@@ -25,15 +28,22 @@ formats =
   # The SDMX-JSON 1.0.0 data format.
   SDMX_JSON_1_0_0: 'application/vnd.sdmx.data+json;version=1.0.0'
 
+  # The SDMX-JSON 2.0.0 data format.
+  SDMX_JSON_2_0_0: 'application/vnd.sdmx.data+json;version=2.0.0'
+
   # The SDMX-CSV 1.0.0 data format, with default labels and
   # non-normalized periods
   SDMX_CSV_1_0_0: 'application/vnd.sdmx.data+csv;version=1.0.0'
 
+  # The SDMX-CSV 1.0.0 data format, with default labels,
```

Comment:
Comment refers to SDMX-CSV 1.0.0 (needs to change to 2.0.0)

Reply:
Thanks, @stratosn, I have updated the PR accordingly